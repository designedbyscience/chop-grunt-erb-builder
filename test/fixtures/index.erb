<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<%= include "header" %>		
		
		Testing
	</body>
</html>
