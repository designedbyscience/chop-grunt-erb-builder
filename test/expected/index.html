<!DOCTYPE html>
<html>
	<head></head>
	<body>
		Hello there!		
		
		Testing
	</body>
</html>
